[2022-11-20T00:45:13.912+0000] {taskinstance.py:1165} INFO - Dependencies all met for <TaskInstance: Call_Center_ETL_DAG.insert_call_info_data manual__2022-11-20T00:44:38.455644+00:00 [queued]>
[2022-11-20T00:45:13.946+0000] {taskinstance.py:1165} INFO - Dependencies all met for <TaskInstance: Call_Center_ETL_DAG.insert_call_info_data manual__2022-11-20T00:44:38.455644+00:00 [queued]>
[2022-11-20T00:45:13.948+0000] {taskinstance.py:1362} INFO - 
--------------------------------------------------------------------------------
[2022-11-20T00:45:13.950+0000] {taskinstance.py:1363} INFO - Starting attempt 1 of 1
[2022-11-20T00:45:13.952+0000] {taskinstance.py:1364} INFO - 
--------------------------------------------------------------------------------
[2022-11-20T00:45:14.105+0000] {taskinstance.py:1383} INFO - Executing <Task(MySqlOperator): insert_call_info_data> on 2022-11-20 00:44:38.455644+00:00
[2022-11-20T00:45:14.128+0000] {standard_task_runner.py:55} INFO - Started process 3547 to run task
[2022-11-20T00:45:14.166+0000] {standard_task_runner.py:82} INFO - Running: ['***', 'tasks', 'run', 'Call_Center_ETL_DAG', 'insert_call_info_data', 'manual__2022-11-20T00:44:38.455644+00:00', '--job-id', '143', '--raw', '--subdir', 'DAGS_FOLDER/call_center_dag.py', '--cfg-path', '/tmp/tmpu45a3v4q']
[2022-11-20T00:45:14.168+0000] {standard_task_runner.py:83} INFO - Job 143: Subtask insert_call_info_data
[2022-11-20T00:45:14.844+0000] {task_command.py:376} INFO - Running <TaskInstance: Call_Center_ETL_DAG.insert_call_info_data manual__2022-11-20T00:44:38.455644+00:00 [running]> on host 9eba6bc17a61
[2022-11-20T00:45:15.687+0000] {taskinstance.py:1592} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=Abdelrhman Yassein
AIRFLOW_CTX_DAG_ID=Call_Center_ETL_DAG
AIRFLOW_CTX_TASK_ID=insert_call_info_data
AIRFLOW_CTX_EXECUTION_DATE=2022-11-20T00:44:38.455644+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=manual__2022-11-20T00:44:38.455644+00:00
[2022-11-20T00:45:15.695+0000] {mysql.py:84} INFO - Executing: 
        INSERT INTO call_info(call_id,reason,channel,call_center,response_time)
        SELECT DISTINCT
            id as call_id,
            reason,
            channel,
            call_center,
            response_time
        FROM 
            staging_call
        WHERE
            id IS NOT NULL;
            
[2022-11-20T00:45:15.851+0000] {base.py:71} INFO - Using connection ID 'mysql_conn_id' for task execution.
[2022-11-20T00:45:16.137+0000] {sql.py:315} INFO - Running statement: 
        INSERT INTO call_info(call_id,reason,channel,call_center,response_time)
        SELECT DISTINCT
            id as call_id,
            reason,
            channel,
            call_center,
            response_time
        FROM 
            staging_call
        WHERE
            id IS NOT NULL;
            , parameters: None
[2022-11-20T00:45:20.724+0000] {sql.py:324} INFO - Rows affected: 32941
[2022-11-20T00:45:21.158+0000] {taskinstance.py:1406} INFO - Marking task as SUCCESS. dag_id=Call_Center_ETL_DAG, task_id=insert_call_info_data, execution_date=20221120T004438, start_date=20221120T004513, end_date=20221120T004521
[2022-11-20T00:45:21.243+0000] {local_task_job.py:159} INFO - Task exited with return code 0
[2022-11-20T00:45:21.291+0000] {taskinstance.py:2623} INFO - 0 downstream tasks scheduled from follow-on schedule check
